apps/show.py

```diff
@@ -89,7 +120,17 @@ def next_packet(self):
                 bytes([packet_type]) + self.source.read(included_length),
             )
 
-        return (packet_flags & 1, self.source.read(included_length))
+        return (ts_dt, packet_flags & 1, self.source.read(included_length))
+
+
+# -----------------------------------------------------------------------------
+class Printer:
+    def __init__(self):
+        self.index = 0
+
+    def print(self, message: str) -> None:
+        self.index += 1
+        print(f"[{self.index}]{message}")
```

Comment:
nit: Could align here to make it prettier 😃 
```suggestion
        print(f"[{self.index:8}]{message}")
```

Reply:
done